src/test/resources/META-INF/mods.toml

```diff
@@ -268,6 +268,8 @@ modId="custom_item_decorations_test"
 modId="item_stacked_on_other_test"
 [[mods]]
 modId="ambient_occlusion_elements_test"
+[[mods]]
+modId="item_prevent_pickup_test"
 
 
-# ADD ABOVE THIS LINE
+# ADD ABOVE THIS LINE
```

Comment:
whitespace change

Reply:
Ah I missed a file that had a new line removed, I will add that back real quick